### ISSUE TITLE
Add option to ignore reads containing "N" (unknown/any nucleotide) 

### ISSUE THREAD

**Issue by boulund** [dib-lab/sourmash#192]
Is it possible to add an option to ignore reads containing "N" (unknown/any nucleotide)? I know I should probably remove those reads, but maybe it could be useful anyway?

**Comment by betatim** [comment]
If you run sourmash with the `-f` option it will [skip kmers](https://github.com/dib-lab/sourmash/blob/6609b3a42b233e8c488d9e340cf79ca7086c5dce/sourmash_lib/kmer_min_hash.hh#L119) that contain a character that isn't one of ACGT. Does that work for you?

**Comment by boulund** [comment]
Sounds great. I misunderstood what the `--force` option would do from the help text. You can close this if you want to.

**Comment by betatim** [comment]
Which sub-command did you look at? I just looked at `sourmash compute -h` which doesn't say anything about what `-f` does. I'll keep this open as a reminder to fix that.

**Comment by boulund** [comment]
Actually never looked at the command line help from the version I was using (oops). I was reading along in the documentation (https://sourmash.readthedocs.io/en/latest/command-line.html)

```
--force -- recompute existing signatures; convert non-DNA characters to N
```

**Comment by betatim** [comment]
Actually, sorry I confused force and force ... what you are looking for is `--check-sequence`

**Comment by betatim** [comment]
#172 fixes the lack of documentation

**Comment by ctb** [comment]
Closed by #240.
